src/main/java/nl/thehyve/ocdu/factories/PatientDataFactory.java

```diff
@@ -82,4 +81,30 @@ private void setValue(String[] row, Map<String, Integer> columnsIndex, String co
     }
 
 
+    public List<String> generatePatientRegistrationTemplate(MetaData metadata, Map<String, String> subjectMap, boolean registerSite) {
+        List<String> result = new ArrayList<>();
+        String header = "Study Subject ID\t";
```

Comment:
Instead of adding strings in this way you could use:
 
        String.join(delimiter, elements);
where delimiter is in your case "\t" and elements are required headers